source/macos/AdaptiveCards/AdaptiveCards/Renderers/BaseCardElementRenderer.swift

```diff
@@ -24,7 +24,9 @@ class BaseCardElementRenderer {
             }
         }
         
-        view.identifier = NSUserInterfaceItemIdentifier(element.getId() ?? "")
+        if let id = element.getId(), !id.isEmpty {
+            view.identifier = NSUserInterfaceItemIdentifier(id)
+        }
         
```

Comment:
Since this alters the basic case for input toggle, can we run the btp for input toggle once